doc/source/api.rst

```diff
@@ -2657,6 +2657,7 @@ objects.
    api.extensions.register_index_accessor
    api.extensions.ExtensionDtype
    api.extensions.ExtensionArray
+   arrays.NumPyExtensionArray
```

Comment:
This is where https://github.com/pandas-dev/pandas/pull/23581 is collecting our arrays.